GameData/RealismOverhaul/RO_SuggestedMods/SOCK/RO_ShuttleOV.cfg

```diff
@@ -0,0 +1,622 @@
+// Shuttle Orbiter Construction Kit
+
+// Orbiter configs
+@PART[benjee10_shuttle_forwardFuselage]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.81
```

Comment:
The rescaleFactor and tempeature manipulations are mostly shared, I suppose they could be put into a single patch for all the parts if you want to cut down the patch size a bit.